content/en/docs/tasks/manage-hugepages/scheduling-hugepages.md

```diff
@@ -7,7 +7,7 @@ description: Configure and manage huge pages as a schedulable resource in a clus
 ---
 
 <!-- overview -->
-{{< feature-state state="stable" >}}
+{{< feature-state feature_gate_name="HugePage" >}}
```

Comment:
Preview build [failed](https://app.netlify.com/sites/kubernetes-io-main-staging/deploys/665fc37467583100089336cc) as the feature gate name needs correction; its `HugePages` ([reference here](https://github.com/kubernetes/website/blob/main/content/en/docs/reference/command-line-tools-reference/feature-gates/huge-pages.md?plain=1#L3)).

```suggestion
{{< feature-state feature_gate_name="HugePages" >}}
```